src/types/index.ts

```diff
@@ -20,16 +20,25 @@ export interface EnvironmentProps {
   environment?: PresetsType | string;
 }
 
+/**
+ * Three point spotlight setup with an additional body lift light.
+ */
 export interface LightingProps {
-  ambientLightColor?: Color | string;
-  ambientLightIntensity?: number;
-  dirLightPosition?: Vector3;
-  dirLightColor?: Color | string;
-  dirLightIntensity?: number;
-  spotLightPosition?: Vector3;
-  spotLightColor?: string;
-  spotLightAngle?: number;
-  spotLightIntensity?: number;
+  keyLightIntensity?: number;
+  keyLightColor?: string; // css-color string
```

Comment:
css-color string? is the same as hexadecimal color?

Reply:
I think storybook provides this in #000000 where the three js color prefers 0x00000 but both should work :)